### ISSUE TITLE
[Packaging] Packaged executable build the 'ontological trees ' at each startup

### ISSUE THREAD

**Issue by pafonta** [BlueBrain/neurocurator#49]
When not packaged, NeuroCurator build the 'ontological trees' only at the first start.

Doing it at each startup slow down the starting of NeuroCurator.

Terminal output:

```
Building ontological tree for  PATO:0000261 ... 
Building ontological tree for  NIFORG:birnlex_569 ... 
Building ontological tree for  CHEBI:23367 ... 
Building ontological tree for  GO:0097458 ... 
Building ontological tree for  NIFGA:birnlex_1167 ... 
Building ontological tree for  NIFMOL:nifext_2506 ... 
Building ontological tree for  NIFSUB:sao914572699 ... 
Building ontological tree for  nifext_8054 ... 
Building ontological tree for  sao1813327414 ... 
```

**Comment by pafonta** [comment]
Should be fixed by #54.

**Comment by pafonta** [comment]
No more relevant after the new releasing process which came with v0.4.1:
https://github.com/BlueBrain/neurocurator/tree/v0.4.1#getting-started
